Simplified/NYPLBookCell.h

```diff
@@ -2,10 +2,9 @@
 // simply to encapsulate dequeueing cells from a UICollectionView so that the logic does not need to
 // be repeated for every part of the application that displays book cells.
 
-#import "NYPLBookState.h"
-
 @class NYPLBook;
 @class NYPLBookCell;
+typedef NS_ENUM(NSInteger, NYPLBookState);
```

Comment:
you should be able to remove this forward declaration: I don't see `NYPLBookState` being used in this header file